src/apis/status.ts

```diff
@@ -44,12 +162,138 @@ function logProjectInfo() {
  *   - If there's a project configuration that contains remote network, show
  *     its status.
  */
-function logNetworkInfo() {
-  if (false) {
-    //
-  } else {
-    console.log(chalk.cyan('\n[Network]'));
-    console.log('No active local network');
-    console.log('You may start the local network with "boson start"');
-  }
+function getAliveNodesTask(): Listr.ListrTask {
+  return {
+    title: 'Getting network information',
+    skip: ctx => {
+      const envConfig = ctx.projConfig.envs[ctx.env];
+      if (typeof envConfig.monitorPort === 'undefined') {
+        throw new Error('Double monitor port is not specified!');
```

Comment:
Hmm I see, so would the monitor process get started by the first `double` command the user runs, and then just stays alive for as long as possible? Does the same monitor watch processes for all the different projects there might be?

Reply:
There's also the issue of assigning RPC ports to local instances. I don't want to put them in the configuration because that feels too bloated, but it's unclear how to handle it transparently across projects.